apps/admin/src/stores/useNotificationStore.ts

```diff
@@ -17,11 +17,10 @@ export const useNotificationStore = defineStore('notifications', () => {
     /**
      * Add a notification to the store.
      */
-    add(notification: PdkNotification, tags?: object | Record<string, unknown>) {
+    add(notification: PdkNotification, tags?: object | Record<string, string>) {
```

Comment:
die object in de type kan wel weg denk ik (moet je wel even checken)

Reply:
Heb het even gecheckt en object moet wel blijven staan